libs/community/langchain_community/llms/bedrock.py

```diff
@@ -93,11 +107,26 @@ class LLMInputOutputAdapter:
 
     @classmethod
     def prepare_input(
-        cls, provider: str, prompt: str, model_kwargs: Dict[str, Any]
+        cls,
+        provider: str,
+        model_kwargs: Dict[str, Any],
+        prompt: Optional[str] = None,
+        system: Optional[str] = None,
+        messages: Optional[List[Dict]] = None,
     ) -> Dict[str, Any]:
         input_body = {**model_kwargs}
         if provider == "anthropic":
-            input_body["prompt"] = _human_assistant_format(prompt)
+            if messages:
+                input_body["anthropic_version"] = "bedrock-2023-05-31"
```

Comment:
is there a plan for some way to set the version as a range - or get the appropriate version from bedrock or anthropic? Only this hard-coding of a specific api version is a bit fragile.
(Although I _am_ anxious to see the PR go ahead so I can use it :))

Reply:
currently this is the only available "anthropic_version"

Reply:
Right, agree on this a bit fragile in there, but that's what Bedrock has so far. There is no documentation on providing any range or getting this info from the Bedrock API.